pos.lua

```diff
@@ -22,27 +22,27 @@ local function posLimit(pos)
 end
 
 minetest.register_chatcommand("select_area", {
-	params = "<ID>",
-	description = "Select a area by id.",
+	params = S("<ID>"),
+	description = S("Select a area by id."),
```

Comment:
While you're at it..
```suggestion
	description = S("Select an area by ID."),
```